lib/ex_unit/test/ex_unit_test.exs

```diff
@@ -138,4 +138,21 @@ defmodule ExUnitTest do
       assert ExUnit.run == %{failures: 0, skipped: 0, total: 1}
     end) =~ "1 test, 0 failure"
   end
+
+  test "it produces error on not implemented tests" do
+    capture_io :stderr, fn ->
```

Comment:
you are right, this is not needed. I have copied over test example from the same file, where, btw, it is not needed too. I can make second PR to clean this up.